templates/web/base/waste/bin_days.html

```diff
@@ -96,7 +96,7 @@ <h3>More services</h3>
             [% IF any_request_allowed %]
               <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
             [% END %]
-            [% IF !pending_subscription AND !services_available.Garden_Waste.is_active %]
+            [% IF NOT waste_features.garden_disabled AND NOT pending_subscription AND NOT services_available.Garden_Waste.is_active %]
```

Comment:
garden_disabled is never set here if there isn't a garden subscription so this will still show.

Reply:
Oh yes, good point